content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -164,16 +164,25 @@ Pod onto Nodes, to restart the daemon Pod after failure, and to help automate up
 
 ## API compatibility
 
-Kubernetes device plugin support is in beta. The API may change before stabilization,
-in incompatible ways. As a project, Kubernetes recommends that device plugin developers:
+Prior to v1.10, the versioning scheme required the Device Plugin's API version to
+match exactly the Kubelet's version. On graduation of this feature to Beta in v1.12
+this is no longer a hard requirement. The API is versioned and has been stable since
+Beta graduation of this feature. Because of this, Kubelet upgrades should be seamless
+but there still may be changes in the API before stabilization making upgrades not
+guaranteed to be non-breaking.
 
-* Watch for changes in future releases.
+NOTE: Graduation of Device Manager to a stable feature in v1.26 does not imply
+graduation of device plugin API to a stable API.
```

Comment:
This should be something like:
```markdown
{{< caution >}}
Although the Device Manager component of Kubernetes is a generally available feature,
the _device plugin API_ is not stable. For information on the device plugin API and
version compatibility, read [Device Plugin API versions](/docs/reference/node/device-plugin-api-versions/).
{{< caution >}}
```
:warning: To make that text make sense, you (or another contributor) must write a PR that adds
the page https://k8s.io/docs/reference/node/device-plugin-api-versions/

Reply:
Ack, will fix. Thanks